library/Asana/App.hs

```diff
@@ -33,30 +28,37 @@ import Options.Applicative
   , progDesc
   , strOption
   )
+import RIO.Text (Text)
+import qualified RIO.Text as T
 import System.Environment (getEnv)
-import System.IO (BufferMode(..), hSetBuffering, stderr, stdout)
+import System.IO (stderr)
 
 data App = App
   { appProjectId :: Text
   , appApiAccessKey :: Text
   , appLogLevel :: LogLevel
   , appPerspective :: Perspective
   , appIgnoreNoCanDo :: Bool
+  , logFunc :: LogFunc
   }
 
+instance HasLogFunc App where
+  logFuncL = lens logFunc (\app logFunc -> app {logFunc})
+
 data Perspective = Pessimistic | Optimistic
 
-type AppM = ReaderT App (LoggingT IO)
+type AppM = RIO App
 
 runApp :: App -> AppM a -> IO a
 runApp app action = do
-  -- Ensure output is streamed if in a Docker container
-  hSetBuffering stdout LineBuffering
-  hSetBuffering stderr LineBuffering
```

Comment:
Oh wow, does RIO do this for us? I've been keeping it in my own apps.

Reply:
Honestly I'm not sure, but this only ever gets run by users, so it doesn't really need it.